ddprof-test/src/test/java/com/datadoghq/profiler/wallclock/ContendedWallclockSamplesTest.java

```diff
@@ -51,13 +51,18 @@ public void after() {
     @TestTemplate
     @ValueSource(strings = {"vm", "vmx", "fp", "dwarf"})
     public void test(@CStack String cstack) {
-        String config = System.getProperty("ddprof_test.config");
-        boolean isSanitizer = config.endsWith("san");
-        boolean isJvmci = System.getProperty("java.vm.version", "").contains("jvmci");
-        assumeFalse(Platform.isZing() || Platform.isJ9());
+        // Skip test entirely on unsupported JVMs (don't use assumeFalse which gets retried)
```

Comment:
> don't use assumeFalse which gets retried

Ouch. This should not be the case. I will take a look at the `@TestTemplate` engine ...